AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -354,6 +354,34 @@ - (void)setNeedsLayout
   }
 }
 
+- (void)layoutIfNeeded
```

Comment:
I totally agree and I removed it. It add's overhead we actually don't need at the moment. It can be called on the view anyway: `[node.view layoutIfNeeded]`.